lib/tasks/change_allow_web_publish_to_false.rake

```diff
@@ -1,11 +1,33 @@
+require "goodcity/rake_logger"
 #rake goodcity:change_allow_web_publish_to_false
-
 namespace :goodcity do
   desc 'change allow_web_publish to false if package.quantity = 0'
   task change_allow_web_publish_to_false: :environment do
     packages = Package.where("quantity = ? and allow_web_publish = ?", 0, true)
+    # code to create log for the rake
```

Comment:
looking much better, can we remove unnecessary log output and no need for 'running rake task ...' now as we prefix each line with the task name in the logger now. so something like:

    log = Goodcity::RakeLogger.new("change_allow_web_publish_to_false")
    log.info("Number of Packages to be updated =#{packages.count}")
    log.debug("allow_web_publish of 1st Package =#{packages.first.allow_web_publish}")
    log.debug("allow_web_publish of last Package =#{packages.last.allow_web_publish}")

Note use of debug here for more verbose output which won't exist in production.